### PR TITLE
Ticket 49802: Problem with purchasing module display and Issue 49739: Remove use of org.apache.commons.net.ntp.TimeStamp

### PR DESCRIPTION
**Pull request by labkey-bpatel**
#### Rationale
Ticket [49802](https://www.labkey.org/WNPRC/support%20tickets/issues-details.view?issueId=49802): Problem with purchasing module display.
Since we are using styling from labkey-ui-components, an update to panel.scss caused this regression.

#### Changes
* Update Purchasing Request form to utilize an additional class 'domain-form-panel' to resolve the issue.
* Update labkey-ui-component version to match release23.11 version in package.json.
* Not related to this particular issue, but include in this PR is: _Issue [49739](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=49739): Remove use of org.apache.commons.net.ntp.TimeStamp or migrate to Date or a java.sql.Timestamp._

### CODE REVIEW

**Review by labkey-adam — APPROVED** [review]
InvoicedItem.java (#49739) change looks good to me. Thanks!

File: WNPRC_Purchasing/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "@fortawesome/free-solid-svg-icons": "^6.4.2",
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@labkey/api": "1.27.0",
-    "@labkey/components": "2.390.4",
+    "@labkey/components": "2.390.6",
```
Comment (cnathe) [inline WNPRC_Purchasing/package.json]: 10k lines changes in the package-lock.json seems like a lot for a patch version bump of the @labkey/components package. That may very well be the case if this package.json wasn't updated in awhile, but just calling it out as something to double check.

File: WNPRC_Purchasing/src/client/components/DocumentAttachmentPanel.tsx [hunk 2]
```diff
@@ -39,12 +39,12 @@ export const DocumentAttachmentPanel: FC<Props> = memo(props => {
...
             <Panel
-                className="panel panel-default"
+                className={'panel panel-default domain-form-panel'}
                 expanded={true}
                 onToggle={function () {}} // this is added to suppress JS warning about providing an expanded prop without onToggle
             >
-                <div className="bg-primary">
-                    <Panel.Heading>
+                <div>
+                    <Panel.Heading className={'bg-primary'}>
```
Comment (cnathe) [inline WNPRC_Purchasing/src/client/components/DocumentAttachmentPanel.tsx]: just double checking, was this an intentional change? it is unrelated to the domain-form-panel class change, so wanted to check.

**Review by cnathe — APPROVED** [review]

File: WNPRC_Purchasing/package.json (hunk 1 above)
Comment (labkey-bpatel) [reply WNPRC_Purchasing/package.json]: good point, reverted this change in both package.json and package-lock.json

**Pull request merged**